src/python/director/teleoppanel.py

```diff
@@ -1134,17 +1134,23 @@ def buildTabWidget(self, jointGroups):
 
             if groupName == 'Neck':
                 def onSendNeckJointPositionGoal():
-                    msg = lcmbotcore.robot_state_t()
-                    msg.num_joints = len(joints)
-                    msg.joint_name = joints
-                    msg.joint_velocity = [0] * len(joints)
-                    msg.joint_effort = [0] * len(joints)
-                    msg.joint_position = [0] * len(joints)
-                    for i, jointName in enumerate(joints):
-                        jointIndex = self.toJointIndex(jointName)
-                        msg.joint_position[i] = self.getJointValue(jointIndex)
-
-                    lcmUtils.publish("JOINT_POSITION_GOAL", msg)
+                    if 'neck_ay' in joints:  # Atlas
+                        neckPitchDegrees = self.getJointValue(self.toJointIndex('neck_ay'))
```

Comment:
variable name is wrong, it is not degrees here, which is ok, the message takes radians